engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/plugin/FrontendPluginConfiguration.scala

```diff
@@ -0,0 +1,12 @@
+package pl.touk.nussknacker.engine.plugin
+
+import argonaut.Json
+import com.typesafe.config.Config
+
+trait FrontendPluginConfigurationProvider {
+
+  def createConfigurations(config: Config): List[FrontendPluginConfiguration]
```

Comment:
why not three methods instead of one method with three fields?

Reply:
I refactored it - now it has more methods